Resources/Prototypes/Entities/Clothing/Head/eva-helmets.yml

```
@@ -52,15 +52,17 @@
 
 #Paramedic Void Helmet
 - type: entity
-  parent: ClothingHeadEVAHelmetBase
+  parent: ClothingHeadHardsuitWithLightBase # goobstation - why didnt this have a light
```

yamllint (1.35.1)

[warning] too few spaces before comment

(comments)